[RFR] Initial commit

djhi opened the pull request.

djhi committed:
[RFR] Initial commit

djhi committed:
fix package.json

djhi committed:
readme

fzaninotto commented on src/index.js:
```diff
@@ -0,0 +1,98 @@
+export default (source, value) => Component => {
+    // Mimic the addLabel functionality of the Admin-on-rest FormField component
+    // Otherwise, when the component is hidden, its label would be displayed anyway
+    const addLabel = Component.defaultProps.addLabel;
```
I'm surprised that you don't have to do the same with `addField`. Did you try with form validation? You should get errors like "Trying to access property `map` on a non-object"

fzaninotto commented on README.md:
````diff
@@ -1,2 +1,162 @@
+);
+```
+
+All powers! Check wether the current full record matches your constraints:
````
s/wether/whether/

fzaninotto reviewed (changes requested):
I'm wondering if a real component wouldn't be better than a HOC for that job. Looking at your examples, having to define a custom component to use the feature is a pain IMO.

sedy-bot committed:
Typo fix s/wether/whether/

As requested by fzaninotto at https://github.com/marmelab/aor-dependent-input/pull/1#discussion_r116456765

djhi committed:
review

djhi committed:
fix package.json description
